data/synthetic_lb_data/data.0.json

```diff
@@ -1 +1 @@
-{"type":"LBDatafile","phases":[{"id":0,"tasks":[{"entity":{"id":0,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":1.0},{"entity":{"id":1,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":0.5},{"entity":{"id":2,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":0.5},{"entity":{"id":3,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":0.5}],"communications":[{"type":"SendRecv","to":{"type":"object","id":5},"messages":1,"from":{"type":"object","id":0},"bytes":2.0},{"type":"SendRecv","to":{"type":"object","id":4},"messages":1,"from":{"type":"object","id":1},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":2},"messages":1,"from":{"type":"object","id":3},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":8},"messages":1,"from":{"type":"object","id":3},"bytes":0.5}]}]}
\ No newline at end of file
+{"type":"LBDatafile","phases":[{"id":0,"tasks":[{"entity":{"seq_id":0,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":1.0},{"entity":{"seq_id":1,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":0.5},{"entity":{"seq_id":2,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":0.5},{"entity":{"seq_id":3,"home":0,"type":"object","migratable":true},"node":0,"resource":"cpu","time":0.5}],"communications":[{"type":"SendRecv","to":{"type":"object","id":5},"messages":1,"from":{"type":"object","id":0},"bytes":2.0},{"type":"SendRecv","to":{"type":"object","id":4},"messages":1,"from":{"type":"object","id":1},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":2},"messages":1,"from":{"type":"object","id":3},"bytes":1.0},{"type":"SendRecv","to":{"type":"object","id":8},"messages":1,"from":{"type":"object","id":3},"bytes":0.5}]}]}
```

Comment:
The tasks field is using `seq_id` but communications still has `id`. I think it should all be `seq_id`.

Reply:
Fixed